pkg/udev/mockdata.go

```diff
@@ -140,7 +142,10 @@ func OsDiskName() (string, string, error) {
 
 // getSyspathOfOsDisk returns syspath of os disk in success
 func getSyspathOfOsDisk(osDiskName string) (string, error) {
-	data, err := ioutil.ReadFile("/sys/class/block/" + osDiskName + "/dev")
+	basePath := "/sys/class/block/"
+	filePath := "/dev"
+	filePath = path.Join(basePath, filePath.Clean(filePath))
+	data, err := ioutil.ReadFile(basePath + osDiskName + filePath)
```

Comment:
it seems clearer to me to do the path.Join and Clean inside ReadFile()

Reply:
```suggestion
	data, err := ioutil.ReadFile(basePath + osDiskName + path.Join(basePath, filePath.Clean(filePath)))
```

do you mean this @richardelling

Reply:
This still seems wrong to me. AIUI, the file to be read is `/sys/class/block/nvme0n1/dev` So I'm expecting something like:
```
basePath := "/sys/class/block"
filePath := "dev"
data, err := ioutil.ReadFile(filepath.Clean(path.Join(basePath, osDiskName, filePath)))
```

Reply:
thanks @richardelling made the changes

Reply:
@soniasingla Can you do the same set of changes in the other 3 files also?